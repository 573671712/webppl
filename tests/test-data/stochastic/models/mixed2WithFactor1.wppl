var model = function() {
  var x = flip();
  var y = gaussian(1, x ? 0.5 : 3);
  factor(x ? -2 : 0);
  return y;
};
