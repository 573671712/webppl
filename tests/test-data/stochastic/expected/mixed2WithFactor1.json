{
  "mean": 1,
  "std": 2.821
}
